GetStream/stream-chat-android
# [CAS-1448] Extract markdown support in separate module

https://stream-io.atlassian.net/browse/CAS-1448

### 🎯 Goal

To reduce the size of SDK and allow users to use custom transformers for any transformation to message text.

### 🛠 Implementation details

- New module `stream-chat-android-markdown-transformer` is added. It provides a markdown implementation of `ChatMessageTextTransformer`.
- `messageTextTransformer` property is added to `ChatUI`.

### 🧪 Testing

- Check if the sample app works exact the same as before. 
- Try to provide your own implementation of `ChatMessageTextTransformer`.

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
